iwyu_tool.py

```diff
@@ -86,10 +86,22 @@ def clang_formatter(output):
     'clang': clang_formatter
 }
 
+
 def get_output(cwd, command):
     """ Run the given command and return its output as a string. """
+    # The environment dictionary handling taken from
+    # https://stackoverflow.com/a/4453495 .
```

Comment:
No space before '.'

Reply:
okay, i rephrased to avoid having the link at the end of the sentence and have the punctuation get merged with the url